docs/user-guide/recipes.rst

```diff
@@ -410,10 +416,10 @@ If we have a tag named latest and it points to the first manifest with digest
 sha256:4ecca..., we can point it to the second manifest with the following
 command::
 
-    $ pulp-admin docker repo tag --repo-id busybox --tag-name latest --manifest-digest sha256:c152ddeda2b828fbb610cb9e4cb121e1879dd5301d336f0a6c070b2844a0f56d
+    $ pulp-admin docker repo tag --repo-id busybox --tag-name latest --manifest-digest sha256:c152ddeda2b828fbb610cb9e4cb121e1879dd5301d336f0a6c070b2844a0f56d --schema-version 1
```

Comment:
i think i will change so we would retrieve the schema_version on server side. it does not make sense to force user to provide schema version when he provided the digest which is the unique identifier for the manifest.